tf-deploy/myvpc/main.tf

```diff
@@ -0,0 +1,43 @@
+# deploys the dev vpc infra resources of the generic service platform
+# See README.md for details on how to fill in all the commented out values.
+
+provider "aws" {
+  region = "us-east-1"
+}
+
+// Note: as a best practice, we recommend using dynamodb for state locking but
+// for simplicity we skip that here.
+terraform {
+  backend "s3" {
+    #bucket         = "CHANGEME"
+    key            = "ecs-example-vpc/terraform/vpc/terraform.tfstate"
+    region         = "us-east-1"
+  }
+
+  required_version = "~> 0.11.7"
+}
+
+module "vpc" {
+  source   = "../../templates/vpc"
+  vpc_name = "ecs-example-vpc"
+  # create this key pair in the AWS console
+  # https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html#having-ec2-create-your-key-pair
+  #ec2_key_name = "CHANGEME"
+  asg_size = "2"
+  azs = ["us-east-1a", "us-east-1b", "us-east-1c"] # for the instances
+  region = "us-east-1"
+  cidr = "10.0.0.0/20"
```

Comment:
Since these are the defaults, maybe leave commented out w/ a note to change if necessary.